apps/site/lib/site_web/controllers/customer_support_controller.ex

```diff
@@ -137,6 +138,19 @@ defmodule SiteWeb.CustomerSupportController do
   defp validate_privacy(%{"privacy" => "on"}), do: :ok
   defp validate_privacy(_), do: "privacy"
 
+  @spec validate_photos(map) :: :ok | String.t()
+  defp validate_photos(%{"photos" => photos}) when is_list(photos) do
+    if Enum.all?(photos, &valid_upload?/1), do: :ok, else: "photos"
+  end
+
+  defp validate_photos(_), do: :ok
+
+  @allowed_attachment_types ~w(image/bmp image/gif image/jpeg image/png image/tiff image/webp)
```

Comment:
we customarily put module attributes at the head of the file, right after the `use/require/import/alias` stuff--in this case that would go on line 4, after the line reading `require Logger` but before the first call to `plug`

Reply:
Makes sense! Wasn't sure what the preference was — I'd have expected Credo to complain, but maybe it doesn't handle this yet.